### PR TITLE
Fix plugin call in example

### PR DESCRIPTION
**Pull request by kwevers**
##### SUMMARY

The name of the plugin in the example isn't correct (anymore).

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

community.general.github_app_access_token

##### ADDITIONAL INFORMATION

```paste below
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while templating '{{ lookup('community.general.github_app_token', key_path='<redacted>',  app_id='<redacted>', installation_id='<redacted>') }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: lookup plugin (community.general.github_app_token) not found"}
```

### CODE REVIEW

File: plugins/lookup/github_app_access_token.py [hunk 1]
```diff
@@ -49,7 +49,7 @@
     dest: /srv/checkout
   vars:
     github_token: >-
-      lookup('github_app_token', key_path='/home/to_your/key',
+      lookup('github_app_access_token', key_path='/home/to_your/key',
```
Comment (felixfontein) [inline plugins/lookup/github_app_access_token.py:52]:
```suggestion
      lookup('community.general.github_app_access_token', key_path='/home/to_your/key',
```

**Review by felixfontein — COMMENTED** [review]
Thanks for fixing this! Actually the name should be a FQCN, see my suggestion below:

File: plugins/lookup/github_app_access_token.py (hunk 1 above)
Comment (kwevers) [reply plugins/lookup/github_app_access_token.py]: Updated as requested

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@kwevers thanks a lot for fixing this!

**Pull request merged**